main.go

```diff
@@ -19,6 +19,9 @@ import (
 // Run the resource and datasource generation tool.
 //go:generate go run gen/generator.go
 
+// Format code and cleanup imports
+//go:generate go run -mod=mod golang.org/x/tools/cmd/goimports -w internal/provider/
+
```

Comment:
This can be removed and the following commit of @samiib should be added: https://github.com/CiscoDevNet/terraform-provider-aci/pull/1227/commits/a2bd622a9bc1857ad6fc6500e80bef68d5fbf7b5

Reply:
Perhaps we should create a separate PR for this so all other PRs can rebase that in.